guides/README.md

```diff
@@ -1,6 +1,6 @@
 # Foreman Guides
 
-This is upstream source code of [Red Hat Satellite 6](https://access.redhat.com/documentation/en-us/red_hat_satellite) documentation.
+This is upstream source code of [Red Hat Satellite 6](https://access.redhat.com/documentation/en-us/red_hat_satellite) documentation as well as parts of the [orcharhino documentation](https://docs.orcharhino.com/or/docs/index.html).
```

Comment:
Nitpick: I'd expect capital letter for Orcharhino.

Reply:
This comes from our style guide: we never capitalize "orcharhino".